### PR TITLE
Updates - moves user prompts into tic-tac-toe class

### PR DESCRIPTION
**Pull request by kknagel**

### CODE REVIEW

**Comment by kknagel** [comment]
Play against computer HW#2

File: src/edu/htc/tictactoe/player/HumanPlayer.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    public HumanPlayer(String name, char marker) {
+        super.name = name;
+        super.gameMarker = marker;
+    }
+
+    public int getMove() {
+        int keyboardInput = 0;
+
+        keyboardInput = TicTacToe.getUserNumericResponse("",1,9);
```
Comment (mbMosman) [inline src/edu/htc/tictactoe/player/HumanPlayer.java]: Nice way to reuse some code for getting numbers. 

Ideally, you'd put this into a separate class though... Maybe a UserInput class or something.

File: src/edu/htc/tictactoe/TicTacToe.java [hunk 2]
```diff
@@ -1,9 +1,137 @@
...
+    public static int getUserNumericResponse(String message, int low, int high){
+        int keyboardInput = 0;
+
+        System.out.println(message);
+
+        Scanner scanner = new Scanner(System.in);
+        do {
+            System.out.println("Enter selection " + low + " - " + high);
+
+            try {
```
Comment (mbMosman) [inline src/edu/htc/tictactoe/TicTacToe.java]: Nice work with the exception handling!

**Comment by mbMosman** [comment]
Nice work overall!  Going to try the JUnit Tests?

File: src/edu/htc/tictactoe/player/HumanPlayer.java (hunk 1 above)
Comment (kknagel) [inline src/edu/htc/tictactoe/player/HumanPlayer.java]: I would be happy to. I think I asked about making a helper class and you
said it was not needed. Is that okay to make this new userInput class?

My son actually found a design flaw that is more like a new level of play
or at least a tweak to what exists is that okay to add this change?
Possibly a new class?

Ken

**Pull request closed without merging**